cfgov/regulations3k/tests/test_regdown.py

```diff
@@ -269,6 +269,12 @@ def test_pseudo_form_field_number_of_underscores(self):
                       '<span></span></span>',
                       regdown('Field: _______'))
 
+    def test_section_symbold_with_non_breaking_space(self):
+        self.assertIn('§&#160;1023.4(a)', regdown('§ 1023.4(a)'))
+        self.assertIn('§&#160;1023.4(a)', regdown('§1023.4(a)'))
+        self.assertIn('§&#160;1023.4(a)', regdown('§\t1023.4(a)'))
+        self.assertIn('§&#160;1023.4(a)', regdown('§      1023.4(a)'))
```

Comment:
We have to use the XML entity here because `&nbsp;` isn't an HTML entity, and when we include block references (inline interps), those get parsed as XML because Python Markdown uses ElementTree as its internal representation of the HTML.